src/main/java/view/UserPageImpl.java

```diff
@@ -1,52 +1,173 @@
 package view;
 
 import model.BlindHorizontal;
-import service.HorizontalService;
-import service.HorizontalServiceImpl;
-import service.PriceCatalog;
+import model.BlindVertical;
+import model.Color;
+import model.MountType;
+import service.blindService.DataSaver;
+import service.blindService.HorizontalService;
+import service.blindService.Validator;
+import service.blindService.VerticalService;
+import service.fileService.DataWriter;
+import service.fileService.DataWriterImpl;
+import service.fileService.Property;
 
+import java.io.File;
 import java.util.Scanner;
 
 public class UserPageImpl implements UserPage {
 
-    private PriceCatalog priceCatalogImpl;
     private HorizontalService horizontalServiceImpl;
+    private VerticalService verticalServiceImpl;
+    private Property property;
+    private DataWriter dataWriter = new DataWriterImpl();
+    private DataSaver saver = new DataSaver();
+    private Validator validator = new Validator();
 
-    public UserPageImpl(HorizontalService horizontalServiceImpl, PriceCatalog priceCatalogImpl) {
+    public UserPageImpl(HorizontalService horizontalServiceImpl, VerticalService verticalServiceImpl, Property property) {
         this.horizontalServiceImpl = horizontalServiceImpl;
-        this.priceCatalogImpl = priceCatalogImpl;
+        this.verticalServiceImpl = verticalServiceImpl;
+        this.property = property;
     }
 
     Scanner scanner = new Scanner(System.in);
-    Scanner scannerStr = new Scanner(System.in);
 
     @Override
     public void showBlindCost() {
-        priceCatalogImpl.initDate();
+        File file = new File(property.getFileNameOutput());
+        file.delete();
```

Comment:
Delete the file with the user's results? I think it's better to create a new file with a unique name rather than the same name when you save the user's results. For example, the output file's name might contain current_date_time. In this case don't need to delete it. What do you think?